MovieMVVM/ViewModels/DetailViewModelProtocol.swift

```diff
@@ -0,0 +1,26 @@
+// DetailViewModelProtocol.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Протокол вью-модели экрана с выбранным фильмом
+protocol DetailMovieViewModelProtocol {
+    // MARK: - Public Properties
+
+    var networkService: NetworkServiceProtocol { get set }
+    var imageService: LoadImageProtocol { get set }
+    var similarMovies: [SimilarMovie] { get set }
+    var movie: Movie { get set }
+    var posterPath: String { get set }
```

Comment:
```suggestion
    var similarMovies: [SimilarMovie] { get set }
    var movie: Movie { get set }
    var posterPath: String { get set }
```
из вьюмодели торчат все эти свойства, но все ли должны быть изменяемыми?